srcbook/README.md

```diff
@@ -24,7 +24,7 @@ Srcbook runs locally on your machine as a CLI application with a web interface.
 
 ### Requirements
 
-- Node.js v20+
+- Node.js v18+
```

Comment:
If we do support 18, then we should change CI to run against 18 (earliest version we support) and the current node version? I think it's currently 20 and 22